kotlin-react-router/src/jsMain/generated/react/router/NavigateFunction.kt

```diff
@@ -1,5 +1,8 @@
+@file:JsModule("react-router")
```

Comment:
Module is for `external` only

Reply:
I know, this fix is in the road map of `karakum` converter